Public/Src/App/Bxl/BuildXLApp.cs

```diff
@@ -924,7 +924,9 @@ private AppResult RunWithLoggingScope(Func<PerformanceMeasurement, AppResult> ru
                         translatedLogDirectory,
                         m_configuration.InCloudBuild(),
                         Directory.GetCurrentDirectory(),
-                        m_initialConfiguration.Startup.ConfigFile.ToString(m_pathTable));
+                        m_initialConfiguration.Startup.ConfigFile.ToString(m_pathTable),
+                        OperatingSystemHelper.IsDotNetCore

```

Comment:
This is fine, but did you consider being more specific i.e. "net472" vs "netcoreapp30" vs "netcoreapp31" when it is released?

Reply:
that would be better, agreed.

Will change